rpc/common/service.proto

```diff
@@ -128,3 +128,10 @@ message CVSS {
   double v2_score  = 3;
   double v3_score  = 4;
 }
+
+message CustomResource {
+  string                 type     = 1;
+  string                 filepath = 2;
```

Comment:
Please be aligned with other fields.
https://github.com/aquasecurity/trivy/blob/d1c352461ae658aabe1c026d8c0be4ff1ded6ee7/rpc/common/service.proto#L17
https://github.com/aquasecurity/trivy/blob/d1c352461ae658aabe1c026d8c0be4ff1ded6ee7/rpc/common/service.proto#L23
https://github.com/aquasecurity/trivy/blob/d1c352461ae658aabe1c026d8c0be4ff1ded6ee7/rpc/common/service.proto#L53